docs/game_sdk/Achievements.md

```diff
@@ -18,15 +18,15 @@ You can also mark achievements as `secret` and `secure`. "Secret" achievements w
 
 ###### Achievement Struct
 
-| name           | type    | description                                                                                                                      |
-| -------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| application_id | Int64   | the unique id of the application                                                                                                 |
```

Comment:
Pretty sure `Snowflake` is wrong here; they're not wrapped in a string.

Reply:
Do you have an example achievement object I could add? Looking at some of the return responses (such as [Get Achievement](https://discord.com/developers/docs/game-sdk/achievements#get-achievement)) it looks like a string to me.

Reply:
My guess is it's returned as a string over REST but not over IPC, could be wrong though. ¯\_(ツ)_/¯

Reply:
Interesting. I have zero experience with IPC so I wouldn't know. Perhaps there should be a note somewhere stating this(?)

Reply:
I think when they're returned by Discord, it's just a generic string (hence it can be a Snowflake). But when inputting them (for example `discord::Core::create(123, ...)` requires a literal integer, not a string, so it can technically be 2 types.

Reply:
Might be good to document as snowflake and add a footnote or something.